### ISSUE TITLE
"Add to Keep" not working when using multiple Google accounts 

### ISSUE THREAD

**Issue by claudiodangelis** [claudiodangelis/popup-for-keep#14]
Ref. https://chrome.google.com/webstore/detail/popup-for-keep/fhcmhglnohogibbbpbodmjeggpdlboop/support

**Comment by claudiodangelis** [comment]
An user is reporting:

```
I have 2 Google accounts, one for work and one personal. I use my personal account for Keep, since I have [redacted]. But Popup for Keep by default only shows the Keep connected to my work account (which has no notes). It doesn't seem I can change this. (When I use Keep either in the Chrome app or on keep.google.com, it opens the personal version, as configured.) I am always logged in to both my Google accounts.
```

**Comment by claudiodangelis** [comment]
Merged with #19
